include/flashinfer/attention/scheduler.cuh

```diff
@@ -313,27 +315,24 @@ struct DecodePlanInfo {
         enable_cuda_graph(false),
         split_kv(false) {}
 
-  // convert DecodePlanInfo to std::vector<int64_t>
-  std::vector<int64_t> ToVector() const {
```

Comment:
still using `std::vector` here, but change the logic in source files (`csrc/*.cu`).